src/commands/moderation/modnick.js

```diff
@@ -0,0 +1,133 @@
+const Command = require("../../structures/Command");
+const { MessageEmbed } = require("discord.js");
+const darkpassword = require("generate-password");
+const Logging = require("../../database/schemas/logging");
+
+module.exports = class extends Command {
+  constructor(...args) {
+    super(...args, {
+      name: 'modnick',
+      aliases: ["mod", "modnickname"],
+      description: "Moderate a users nickname to make pinging possible",
+      category: "Moderation",
+      guildOnly: true,
+      botPermission: ["MANAGE_NICKNAMES"],
+      userPermission: ["MANAGE_NICKNAMES"],
+      usage: "<@user> [reason]",
+      examples: "modnick @Peter",
+      cooldown: 30,
+    });
+  }
+
+  async run(message, args) {
+    const client = message.client
+    const logging = await Logging.findOne({ guildId: message.guild.id });
+    
+    const member =
+      message.mentions.members.last() ||
+      message.guild.members.cache.get(args[0]);
+
+    const reason = args.slice(1).join(" ");
+
+    const impostorpassword = darkpassword.generate({
```

Comment:
Don't use an package for an simple random ID String. For Example use this:

`Math.random().toString(36).substring(2, 5) + Math.random().toString(36).substring(2, 5)`